mlir/test/Target/LLVMIR/Import/alias.ll

```diff
@@ -0,0 +1,45 @@
+; RUN: mlir-translate --import-llvm %s | FileCheck %s
```

Comment:
Didn't look at the test in detail yet but we probably want to have some checks for the verifiers in `llvm-project/mlir/test/Dialect/LLVMIR/invalid.mlir`.